chain/gssmr-v3substrate/config.toml

```diff
@@ -0,0 +1,49 @@
+[global]
```

Comment:
Should we move those chains to `chain/v3substrate/...` instead?

Also why is it `v3substrate` btw?? 😄

Reply:
there is a `dev` and `gssmr` v3 substrate.  Elizabeth put in these runtimes from the v3 release commit.  Since then, I they stopped tagging vXXX releases.